torch/csrc/jit/runtime/symbolic_shape_registry_util.cpp

```diff
@@ -0,0 +1,124 @@
+      {"aten::frac(Tensor self) -> Tensor", "unary"},
+      {"aten::__and__.Scalar(Tensor self, Scalar other) -> Tensor", "unary"},
+      {"aten::__or__.Scalar(Tensor self, Scalar other) -> Tensor", "unary"},
+      {"aten::__xor__.Scalar(Tensor self, Scalar other) -> Tensor", "unary"},
+      {"aten::__lshift__.Scalar(Tensor self, Scalar other) -> Tensor", "unary"},
+      {"aten::__rshift__.Scalar(Tensor self, Scalar other) -> Tensor", "unary"},
+      {"aten::where.Scalar(Tensor condition, Scalar self, Scalar other) -> Tensor", "unary"},
+      {"aten::add.Tensor(Tensor self, Tensor other, *, Scalar alpha=1) -> Tensor", "broadcast"},
+      {"aten::type_as(Tensor self, Tensor other) -> Tensor", "broadcast"},
+      {"aten::sub.Tensor(Tensor self, Tensor other, *, Scalar alpha=1) -> Tensor", "broadcast"},
+      {"aten::mul.Tensor(Tensor self, Tensor other) -> Tensor", "broadcast"},
+      {"aten::div.Tensor(Tensor self, Tensor other) -> Tensor", "broadcast"},
+      {"aten::eq.Tensor(Tensor self, Tensor other) -> Tensor", "broadcast"},
+      {"aten::ne.Tensor(Tensor self, Tensor other) -> Tensor", "broadcast"},
+      {"aten::ge.Tensor(Tensor self, Tensor other) -> Tensor", "broadcast"},
+      {"aten::gt.Tensor(Tensor self, Tensor other) -> Tensor", "broadcast"},
+      {"aten::le.Tensor(Tensor self, Tensor other) -> Tensor", "broadcast"},
+      {"aten::lt.Tensor(Tensor self, Tensor other) -> Tensor", "broadcast"},
+      {"aten::lerp.Scalar(Tensor self, Tensor end, Scalar weight) -> Tensor", "broadcast"},
+      {"aten::fmod.Tensor(Tensor self, Tensor other) -> Tensor", "broadcast"},
+      {"aten::atan2(Tensor self, Tensor other) -> Tensor", "broadcast"},
+      {"aten::remainder.Tensor(Tensor self, Tensor other) -> Tensor", "broadcast"},
+      {"aten::__and__.Tensor(Tensor self, Tensor other) -> Tensor", "broadcast"},
+      {"aten::__or__.Tensor(Tensor self, Tensor other) -> Tensor", "broadcast"},
+      {"aten::__xor__.Tensor(Tensor self, Tensor other) -> Tensor", "broadcast"},
+      {"aten::__lshift__.Tensor(Tensor self, Tensor other) -> Tensor", "broadcast"},
+      {"aten::__rshift__.Tensor(Tensor self, Tensor other) -> Tensor", "broadcast"},
+      // TODO: enable other min/max variants, operators that can be both
+      // elementwise or reductions:
+      {"aten::min.other(Tensor self, Tensor other) -> Tensor", "broadcast"},
+      {"aten::max.other(Tensor self, Tensor other) -> Tensor", "broadcast"},
+      {"aten::lerp.Tensor(Tensor self, Tensor end, Tensor weight) -> Tensor", "broadcast_three"},
+      {"aten::addcmul(Tensor self, Tensor tensor1, Tensor tensor2, *, Scalar value=1) -> Tensor", "broadcast_three"},
+      {"aten::where.self(Tensor condition, Tensor self, Tensor other) -> Tensor", "broadcast_three"},
+      {"aten::where.ScalarSelf(Tensor condition, Scalar self, Tensor other) -> Tensor", "broadcast_one_three"},
+      // TODO: enable slice, shape inference is not implemented for this op yet
```

Comment:
The list didn't change during the move, right?

Reply:
yes - just copy-pasting

Reply:
no, ctrl-F "aten::" 98 results in both lists